pom.xml

```diff
@@ -76,6 +76,12 @@
 				<artifactId>logback-json-classic</artifactId>
 				<version>0.1.5</version>
 			</dependency>
+
+			<dependency>
+				<groupId>com.google.guava</groupId>
+				<artifactId>guava</artifactId>
+				<version>20.0</version>
```

Comment:
Looks like there are much newer Guava versions: https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.google.guava%22%20AND%20a%3A%22guava%22

What makes you to stick with this one?

Also bear in mind: we somehow try to stay away of using Guava at all:
https://github.com/spring-projects/spring-boot/commit/654fe9a31c0588cae0cdb5d8301717c3af605fad#diff-7050cd73a8895c61627d3891a0a14bc7R76

We can talk about that tomorrow with Stephane

Reply:
google-cloud-java uses it. https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/pom.xml#L145

I'm still trying to figure out what's forcing guava's version to 18.0..

Reply:
Doesn't `mvnw dependency:tree` help here to find out ?

Reply:
@artembilan in #471 @snicoll seems to agree with this approach. PTAL

Reply:
I think I found the culprit:

https://github.com/spring-cloud/spring-cloud-netflix/blob/master/spring-cloud-netflix-dependencies/pom.xml#L423

It's pulled in via `spring-cloud-dependencies`.

Reply:
Fine with me. Now I understand that the point is like do not use Guava from our code.
And that rule might be applied really only for the Spring Boot project.

Glad to see that you have figured out the culprit!
I wonder why do we need this Netflix dependency? 
Can't we just exclude it for free?